main/acle.md

```diff
@@ -360,9 +360,10 @@ Armv8.4-A [[ARMARMv84]](#ARMARMv84). Support is added for the Dot Product intrin
 * Added the [Keyword attributes](#keyword-attributes) section.
 * Changed the [SME language extensions](#sme-language-extensions-and-intrinsics)
   to use keyword attributes instead of GNU-style attributes.
-* Added missing word to Function Multi Versioning's [Name mangling](#name-mangling).
```

Comment:
I think this was unnecessary. IMO typos and minor syntactive improvements don't need to be release noted. Also I believe grouping all the changes to Function Multi Versioning under the same bulletpoint helps readability. If you insist on keeping it can we please move it down with the rest?

Reply:
That is a good point. Let's follow your suggestion and have it removed.